### ISSUE TITLE
Terrible overfitting in 1d Scattering example

### ISSUE THREAD

**Issue by psorus** [kymatio/kymatio#704]
Simply downloading and running this example from your documentation

[https://www.kymat.io/gallery_1d/classif_keras.html#sphx-glr-gallery-1d-classif-keras-py
](https://www.kymat.io/gallery_1d/classif_keras.html#sphx-glr-gallery-1d-classif-keras-py
)
should reach an accuracy of 98% on test data and 99.7% on training data.
But when I run it, I reach 75% (clearly overfitting) and 99% respectively.

See [https://ibb.co/MZVy6qj](https://ibb.co/MZVy6qj) for my logs(I did not change the code at all), I did try running it on colab to be easier reproducable, but theres the other thing: According to your website, training should take about 6min, but each colab epoch takes ~5min for me (and I dont want to wait another 50*5min>4h)

**Comment by gexarcha** [comment]
Hi, 
your results deviate substantially from the results in https://www.kymat.io/gallery_1d/plot_classif_torch.html
both in terms of time and accuracy. 
Perhaps you can provide more details about your system configuration?
It appears that something is wrong in your setup.
Best,
Georgios

**Comment by psorus** [comment]
thanks for the response, but I dont really think this is a problem with my setup. 

I am using a manjaro thinkpad (python 3.9, tensorflow, keras cpu, installed through aur [https://archlinux.org/packages/community/x86_64/python-tensorflow/](https://archlinux.org/packages/community/x86_64/python-tensorflow/), not changed at all), and I've never had problems with this.

On the other hand, Ive tried it on google colab (which should be configured pretty much perfectly for tensorflow, as it hosted by google directly), and it needs even longer per epoch (For performance it needs to run a lot more).
See [https://colab.research.google.com/drive/1XPodjlpV8Qv-EHL3eajdffk-ERyH-7ZX?usp=sharing](https://colab.research.google.com/drive/1XPodjlpV8Qv-EHL3eajdffk-ERyH-7ZX?usp=sharing)

**Comment by gexarcha** [comment]
For colab try Edit > Notebook Settings > Enable GPU. For the laptop, indeed there should be a performance penalty, this work is typically executed in GPU enabled workstations.

**Comment by psorus** [comment]
Thanks for that. It just finished with 94.6% (test) and 97.7% (training). Still not really what you wrote in your documentation, but way less significant. I guess I have to check my setup
